R/validityChecks.R

```diff
@@ -1000,6 +1000,50 @@
     }
 }
 
+.valid.minDistToCells.input <- function(object,x_cells,name,coords,img_id,return_neg){
+  if(!is(object, "SingleCellExperiment")) {
+    stop("'object' not of type 'SingleCellExperiment'.")
+  }
+  
+  if (!all(is.logical(x_cells))) {
+    stop("'x_cells' must all be logical.")
+  }
+  
+  if (length(x_cells) != ncol(object)) {
+    stop("Length of 'x_cells' must match the number of cells in 'object'.")
+  }
+  
+  if (!is.null(name) & (length(name) != 1 | !is.character(name))) {
```

Comment:
You can remove the `is.null` check here, otherwise the user can suppress an outbut via `name = NULL`.